src/groundlight/experimental_api.py

```diff
@@ -180,16 +181,26 @@ def get_notes(self, detector: Union[str, Detector]) -> Dict[str, Any]:
         det_id = detector.id if isinstance(detector, Detector) else detector
         return self.notes_api.get_notes(det_id)
 
-    def create_note(self, detector: Union[str, Detector], note: Union[str, NoteRequest]) -> None:
+    def create_note(self, detector: Union[str, Detector], note: str, image: Union[str, None] = None) -> None:
```

Comment:
nit: I though `Union` was replaced with `|`, e.g. `detector: str | Detector`. Maybe this doesn't work for some of the python versions we support?